tests/vexflow_test_helpers.ts

```diff
@@ -120,19 +120,7 @@ const SVG_TEST_CONFIG = {
   backend: Renderer.Backends.SVG,
   tagName: 'div',
   testType: 'SVG',
-  fontStacks: [
-    'Bravura',
-    'Finale Ash',
-    'Finale Broadway',
-    'Finale Maestro',
-    'Gonville',
-    'Gootville',
-    'Leland',
-    'Leipzig',
-    'MuseJazz',
-    'Petaluma',
-    'Sebastian',
-  ],
+  fontStacks: ['Bravura'],
```

Comment:
Let's just comment out all the other fonts for now. That will make it easier for developer to know which fonts they can test. (They can un-comment a font if they wish.)

Later on let's figure out how to allow people to select fonts via the UI.

Reply:
Hi Ron I decided to leave two so that the possible collaborators easily see that it is possible to use different fonts.